packages/tasks/src/model-libraries.ts

```diff
@@ -114,6 +114,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"npz"`,
 	},
+	birder: {
```

Comment:
Thanks, just to double confirm are these the only two model checkpoints supported by the tag: https://huggingface.co/models?other=birder

Reply:
Yeah, I'm still testing before uploading more...
I have around 30 more models with support of more geographical regions.

Reply:
Nice! Lovely! Let us know if you need any help.

Reply:
Thanks a lot :)

I'm good for now, documentation is really good.